client/src/featureform/client.py

```diff
@@ -188,6 +188,41 @@ def close(self):
         """
         self.impl.close()
 
+    def columns(
```

Comment:
out of curiosity, what should be the expected behavior when the source hasn't been previously registered?

Reply:
I would expect a `NotFound` error. Does that seem reasonable?